src/clj/rems/db/applications.clj

```diff
@@ -47,8 +47,10 @@
     (format-external-id (allocate-external-id! id-prefix))))
 
 (defn allocate-application-ids! [time]
-  {:application/id (:id (db/create-application!))
-   :application/external-id (application-external-id! time)})
+  (let [external-id (application-external-id! time)]
+    {:application/id (:id (db/create-application!))
+     :application/generated-external-id external-id
```

Comment:
is this really used? application-base-view sets `:application/generated-external-id (:application/external-id event)`

Reply:
I guess this got left over as a half solution. I'll remove this.